docs/src/content/docs/pt-pt/resources/plugins.mdx

```diff
@@ -66,9 +66,76 @@ Estenda o seu site com os plugins oficiais suportados pela equipa Starlight e co
 		<LinkCard
 		href="https://github.com/lorenzolewis/starlight-utils"
 		title="starlight-utils"
-		description="Extenda o Starlight com uma coleção de utilitários comuns."
+		description="Estenda o Starlight com uma coleção de utilitários comuns."
 	/>
+<LinkCard
+		href="https://github.com/trueberryless/starlight-view-modes"
+		title="starlight-view-modes"
+		description="Adicione a capacidade de mostrar diferentes formas de visualização no seu website de documentação."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-versions"
+		title="starlight-versions"
+		description="Acrescente a versão às páginas da sua documentação Starlight."
+	/>
+	<LinkCard
+		href="https://inox-tools.fryuni.dev/star-warp"
+		title="star-warp"
+		description="Acelere a pesquisa de resultados na sua documentação."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-heading-badges"
+		title="starlight-heading-badges"
+		description="Adicione distintivos aos cabeçalhos de Markdown e MDX."
+	/>
+	<LinkCard
+		href="https://github.com/Fevol/starlight-site-graph"
+		title="starlight-site-graph"
+		description="Adicione um mapa interativo do site à barra lateral das suas páginas."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-sidebar-topics"
+		title="starlight-sidebar-topics"
+		description="Divida a sua documentação em secções diferentes cada uma com a sua barra lateral."
+	/>
+	<LinkCard
+		href="https://github.com/trueberryless-org/starlight-sidebar-topics-dropdown"
+		title="starlight-sidebar-topics-dropdown"
+		description="Divida as suas páginas de documentação em várias sub-páginas e alterne entre elas através de um menu na barra lateral."
+	/>
+	<LinkCard
+		href="https://github.com/trueberryless-org/starlight-cooler-credit"
+		title="starlight-cooler-credit"
+		description="Adicione os créditos do Starlight ou Astro no fundo da tabela de conteúdos."
+	/>
+	<LinkCard
+		href="https://github.com/trueberryless-org/starlight-contributor-list"
+		title="starlight-contributor-list"
+		description="Apresente uma lista de todos os contribuidores do seu projeto."
+	/>
+</CardGrid>
+
+### Temas da comunidade
 
+Um Tema é um plugin do Starlight que muda visualmente a aparência do site através da sobreposição de componentes, CSS personalizados, ou outras
+novas funcionalidades.
+
+<CardGrid>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-theme-rapide"
+		title="starlight-theme-rapide"
+		description="Tema do Starlight inspirado no tema Vitesse do Visual Studio Code."
+	/>
+	<LinkCard
+		href="https://github.com/Fevol/starlight-theme-obsidian"
+		title="starlight-theme-obsidian"
+		description="Tema do Starlight inspirado em sites com o tema Obsidian Publish."
+	/>
+	<LinkCard
+		href="https://github.com/TheOtterlord/catppuccin-starlight"
+		title="catppuccin-starlight"
+		description="Tema de `Pastel Suavizante` para o Starlight"
```

Comment:
Just to double check, is the backticks around "Pastel Suavizante" expected?

Reply:
Yes, because it's the name of the Theme.

Reply:
Ah I think that might be a misunderstanding. The name of the theme is “catppuccin”. The English reads:

> Soothing pastel theme for Starlight

Which means more like “A theme for Starlight in soothing pastel colors”

So something like “Tema pastel suave para Starlight” or “Tema suave em tons pastel para Starlight” might be a closer match.